packages/storybook/src/util.ts

```diff
@@ -0,0 +1,17 @@
+export const transformConfigToProps = (config: { inputs: any[] }) => {
```

Comment:
should we call this `getDefaultProps` instead as it seems that is what this does - grabs default props object from inputs?